deployment/terraform/utils.go

```diff
@@ -67,6 +67,14 @@ func (t *Terraform) OpenSSHFor(resource string) error {
 			}
 		}
 	}
+	if cmd == nil {
+		switch resource {
```

Comment:
Can we move this to the upper block?

Reply:
Good suggestion! Changes pushed, along with new documentation for using `ltctl ssh`.